expression/builtin_info.go

```diff
@@ -976,7 +976,7 @@ func (c *formatNanoTimeFunctionClass) getFunction(ctx sessionctx.Context, args [
 		return nil, err
 	}
 	bf := newBaseBuiltinFuncWithTp(ctx, args, types.ETString, types.ETReal)
-	bf.tp.Flag |= mysql.UnsignedFlag
+	bf.tp.Charset = mysql.DefaultCharset
```

Comment:
Why do we modify this instead of adding a new line?

Reply:
We should use the connection charset and collation here.

Reply:
@XuHuaiyu 
```
$ mysql -h127.0.0.1 -P4000 -uroot --column-type-info=1 -e 'SELECT FORMAT_BYTES(1024), CAST(FORMAT_BYTES(1024) as CHAR)'
Field   1:  `FORMAT_BYTES(1024)`
Catalog:    `def`
Database:   ``
Table:      ``
Org_table:  ``
Type:       VAR_STRING
Collation:  utf8_bin (83)
Length:     0
Max_length: 8
Decimals:   31
Flags:      
```

`Flags: ` should be empty, so unset `bf.tp.Flag`

Reply:
@wjhuang2016 
I use `bf.tp.Charset, bf.tp.Collate = ctx.GetSessionVars().GetCharsetInfo()` but seems wrong, UT failed.
What is the meaning of `connection charset and collation`?

Reply:
Because the UT is wrong, the result should be 1.
FYI https://dev.mysql.com/doc/refman/8.0/en/charset-connection.html